src/box/CMakeLists.txt

```diff
@@ -275,6 +275,7 @@ set(box_sources
     sql_stmt_cache.c
     wal.c
     call.c
+    rt_lua_call_access.c
```

Comment:
How about just `rt_access.c`? It seems, it fits good and it is concise.

Reply:
renamed.